overlord/snapstate/aliasesv2.go

```diff
@@ -225,8 +230,8 @@ func refreshAliases(st *state.State, info *snap.Info, curAliases map[string]*Ali
 		if curTarget.Manual == "" {
 			continue
 		}
-		if info.Apps[curTarget.Manual] == nil {
-			// not an existing app
+		if !nonDaemon(info, curTarget.Manual) {
```

Comment:
Is there a way to avoid the double negative here (`!nonDaemon()`). Maybe something like: `if info.Apps[curTarget.Manual] != nil || isDaemon(info, curTarget.Manual)` (slightly longer but I find it easier to read, but maybe thats just a personal preference).